web/src/components/filings-calendar/tax-access-modal/TaxAccessStepTwo.tsx

```diff
@@ -179,6 +199,7 @@ export const TaxAccessStepTwo = (props: Props): ReactElement => {
         setIsLoading(false);
         analytics.event.tax_calendar_modal.submit.tax_deadlines_added_to_calendar();
         props.onSuccess();
+        await updateQueue.queueTaskProgress({ "determine-naics-code": "COMPLETED" }).update();
```

Comment:
we need to use the `useUpdateTaskProgress` here instead so that the section-completion logic gets triggered as needed